OJP/OJP_SituationSupport.xsd

```diff
@@ -31,4 +31,12 @@
 			<xs:extension base="siri:SituationFullRefStructure"/>
 		</xs:complexContent>
 	</xs:complexType>
+	<xs:complexType name="SituationRefList">
+		<xs:annotation>
+			<xs:documentation>List of references to SITUATIONs.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element ref="SituationFullRef" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
If such a optional list (due to `<xs:element name="SituationFullRefs" type="SituationRefList" minOccurs="0">`) is given it should not be empty, correct? So I think it should be `minOccurs="1"` (which is the default, therefore the `minOccurs` should be omitted)

Reply:
changed